scripts/create_universe.sh

```diff
@@ -178,7 +178,6 @@ fi
 # Start the tservers.
 ###############################################################################
 echo "Starting tservers..."
-TSERVER_CONF_CMD="echo '--tserver_master_addrs=${YB_MASTER_ADDRESSES}' >> ${YB_HOME}/tserver/conf/server.conf"
```

Comment:
why are we removing this?, is this duplicate?

Reply:
Yes, its duplicated thats why removed here.